app/src/main/AndroidManifest.xml

```diff
@@ -18,14 +18,27 @@
         <activity
             android:name=".LauncherActivity"
             android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.Dark">
+            android:theme="@style/AppTheme.Dark"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
+        <activity
```

Comment:
I think instead of copying all these activities into our manifest, it's better to just update these test libraries (newest versions already have `android:exported="true"`; in my case I needed to upgrade `androidx.test:core`, `androidx.test:runner`, `androidx.test:rules`, `androidx.fragment:fragment-testing` - all to `1.4.0` and espresso to `3.4.0`)

Reply:
thanks for the idea, fixed